packages/orchestration/test/utils/address.test.ts

```diff
@@ -107,3 +108,40 @@ test('makeICQChannelAddress', t => {
     'makeICQChannelAddress not hardened against malformed version. use `validateRemoteIbcAddress` to detect this, or expect IBC ProtocolImpl to throw',
   );
 });
+
+test('getBech32Prefix', t => {
+  const testCases: (
+    | { specimen: string; expected: string; error?: never }
+    | { specimen: string; error: string; expected?: never }
+  )[] = [
+    {
+      specimen: 'bc1qw508d6qejxtdg4y5r3zarvary0c5xw7kv8f3t4',
+      expected: 'bc',
+    },
+    {
+      specimen: 'cosmos1n4f2eqt2gm5mh6gevf8aw2wrf75q25yru09yvn',
+      expected: 'cosmos',
+    },
+    {
+      specimen: '111qw508d6qejxtdg4y5r3zarvary0c5xw7kv8f3t4',
+      expected: '11',
```

Comment:
:clap: 

This test distinguishes the correct `.lastIndexOf()` implementation from the `indexOf()` style.